build/teamcity/internal/release/build-and-publish-patched-go-fips.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
```

Comment:
Nit: Would be good to have this be part of the same job that builds the rest of the `go` toolchains so we don't have to trigger multiple jobs to upgrade `go`.

Reply:
Agree. I'll merge the scripts when I'm happy with the result.